azext_iot/_params.py

```diff
@@ -761,6 +761,72 @@ def load_arguments(self, _):
             type=int,
             help="Maximum number of configurations to return. By default all configurations are returned.",
         )
+    
+    with self.argument_context("iot hub state export") as context:
+        context.argument(
+            "filename",
+            options_list=["--filename", "-f"],
+            help="The path to the file where the hub and device information will be stored."
+        )
+
+    with self.argument_context("iot hub state import") as context:
+        context.argument(
+            "filename",
+            options_list=["--filename", "-f"],
+            help="The path to the file from which the hub and device information will be read."
+        )
+        context.argument(
+            "overwrite",
+            options_list=["--overwrite"],
+            help="If this flag is set, then the import will overwrite all configurations and devices in the destination hub."
+        )
+
+    with self.argument_context("iot hub state migrate") as context:
+        context.argument(
+            "hub_name",
+            options_list=["--destination-hub", "--dh"],
+            help="Name of IoT Hub to which the origin hub will be copied."
+        )
+        context.argument(
+            "rg",
+            options_list=["--destination-resource-group", "--dg"],
+            help="Name of resource group of the IoT Hub to which the origin hub will be copied."
```

Comment:
Name of resource group of the IoT Hub which the origin hub will be copied to.

^ this may be more grammatically correct. same with from sentence

Reply:
I've had "don't end a sentence with a proposition" drilled into my head all my life but I think either is grammatically correct.

Reply:
lets consult word for grammar